app/assets/javascripts/components/students/assign_button.jsx

```diff
@@ -109,6 +109,9 @@ const AssignButton = React.createClass({
 
     if (assignment.title === '' || assignment.title === 'undefined') {
       return;
+    } else if (assignment.title.length > 255) {
```

Comment:
Here it would be helpful to add a comment explaining why we need to limit the title length.